keps/README.md

```diff
@@ -9,14 +9,16 @@ This process is still in a _beta_ state and is mandatory for all enhancements be
 
 1. Socialize an idea with a sponsoring SIG.
    Make sure that others think the work is worth taking up and will help review the KEP and any code changes required.
-2. Follow the process outlined in the [KEP template](YYYYMMDD-kep-template.md)
+2. Follow the process outlined in the [KEP template](NNNN-kep-template/README.md)
 
 ## FAQs
 
 ### Do I have to use the KEP process?
 
-No... but we hope that you will.
-Over time having a rich set of KEPs in one place will make it easier for people to track what is going in the community and find a structured historic record.
+More or less, yes.
+
+Having a rich set of KEPs in one place will make it easier for people to track
+what is going in the community and find a structured historic record.
 
 KEPs are only required when the changes are wide ranging and impact most of the project.
```

Comment:
Is this still true? Rather than "more or less" perhaps we should say something like:

*Yes* for:
 - Anything that may be controversial
 - Anything new features except the very smallest
 - Any major change to an existing feature
 - Any change that is wide ranging and has an impact on most of the project. These changes are usually coordinated...

Reply:
Agreed. Maybe we should make a list of things you don't have to submit a KEP for?
We can consider that a follow-up.

Reply:
will reword